File: src/main/java/com/asap/server/domain/ConfirmedDateTime.java
package com.asap.server.domain;

import lombok.Getter;

import javax.persistence.Embeddable;
import java.time.LocalDateTime;

@Getter
@Embeddable
public class ConfirmedDateTime {
    private LocalDateTime confirmedStartTime;
    private LocalDateTime confirmedEndTime;
}

File: domain-db/src/main/kotlin/com/asap/server/domain/meeting/ConfirmedDateTime.kt
package com.asap.server.domain.meeting

import jakarta.persistence.Column
import jakarta.persistence.Embeddable
import java.time.LocalDateTime

@Embeddable
data class ConfirmedDateTime(
    @Column(name = "confirmed_start_time")
    val startTime: LocalDateTime,
    @Column(name = "confirmed_end_time")
    val endTime: LocalDateTime
)